.github/workflows/container.yml

```diff
@@ -12,6 +12,9 @@ on:
   workflow_call:
   workflow_dispatch:
   repository_dispatch:
+
+  # Temporary: build on PRs, to make sure workflow succeeds
+  pull_request:
```

Comment:
The comment says "temporary" but I'd still like to merge it: I'll follow up with smaller tweaks in subsequent PRs, after observing how the migrated deploy workflows behave, once they land in main.